packages/discord.js/src/managers/GuildMemberManager.js

```diff
@@ -441,6 +441,34 @@ class GuildMemberManager extends CachedManager {
     return this.guild.bans.remove(user, reason);
   }
 
+  /**
+   * The options for {@link GuildMemberManager#addRole} and {@link GuildMemberManager#removeRole}
+   * @typedef RoleAddOptions
```

Comment:
Let's call this `RoleModifyOptions` since it is used for both adding and removing a role.